Dockerfile

```diff
@@ -13,13 +14,10 @@ COPY swoole.ini /usr/local/etc/php/conf.d/000-swoole.ini
 RUN pecl install swoole-4.0.4 && docker-php-ext-enable swoole
 
 # Install composer
-COPY getcomposer.sh /usr/local/bin/
-RUN /usr/local/bin/getcomposer.sh
+COPY --from=composer /usr/bin/composer /usr/local/bin/composer
 
 # Project files
-RUN mkdir -p /var/www/public
 COPY index.php /var/www/public/
-COPY entrypoint /usr/local/bin/
 
 EXPOSE 9000
-ENTRYPOINT ["entrypoint"]
+ENTRYPOINT ["php","public/index.php"]
```

Comment:
I'm going to revert this. Having an actual script means that users can copy in an entrypoint script to the same location, without needing to change the ENTRYPOINT in the extending container. Changing it would be a BC break currently.